### PR TITLE
Reorder README text 

### PR DESCRIPTION
**Pull request by Shreeshrii**
Add an Issue Template
Reorganize Readme.MD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
 # Tesseract OCR
 
-For the latest online version of the README.md see:
-    
-https://github.com/tesseract-ocr/tesseract/blob/master/README.md
-  
-### Build
```
Comment (egorpugin) [inline README.md]: I want this line back.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,9 @@
...
 
-For the latest online version of the README.md see:
-    
-https://github.com/tesseract-ocr/tesseract/blob/master/README.md
-  
-### Build 
-[![Build Status](https://travis-ci.org/tesseract-ocr/tesseract.svg?branch=master)](https://travis-ci.org/tesseract-ocr/tesseract)
-[![Build status](https://ci.appveyor.com/api/projects/status/miah0ikfsf0j3819/branch/master?svg=true)](https://ci.appveyor.com/project/zdenop/tesseract/)
-
-### Other
```
Comment (egorpugin) [inline README.md]: too

File: README.md (hunk 1 above)
Comment (stweil) [inline README.md]: It was moved to the end of the file.

File: README.md (hunk 2 above)
Comment (stweil) [inline README.md]: They are still there, but with a better text.

**Comment by stweil** [comment]
The new README.md can be seen [here](https://github.com/Shreeshrii/tesseract).

File: README.md [hunk 3]
```diff
@@ -20,48 +13,28 @@ The lead developer is Ray Smith. The maintainer is Zdenko Podobny.
...
 Tesseract was originally developed at Hewlett-Packard Laboratories Bristol and
 at Hewlett-Packard Co, Greeley Colorado between 1985 and 1994, with some
 more changes made in 1996 to port to Windows, and some C++izing in 1998.
-
 In 2005 Tesseract was open sourced by HP. Since 2006 it is developed by Google.
 
-[Release Notes](https://github.com/tesseract-ocr/tesseract/wiki/ReleaseNotes)
-
-# For developers
+The latest stable version is **[3.05.00](https://github.com/tesseract-ocr/tesseract/releases/tag/3.05.00)**, released in February 2017. Source code is available from [3.05 branch on github](https://github.com/tesseract-ocr/tesseract/tree/3.05). 3.05.01 bug-fix release is expected in May/June 2017.
```
Comment (stweil) [inline README.md]: s/on github/on GitHub/

File: README.md (hunk 2 above)
Comment (egorpugin) [inline README.md]: Where? The idea behind this caption was to split badges.
In this PR they are again near each other.

**Comment by Shreeshrii** [comment]
Thanks for your feedback.

I was looking at the Readme from the point of view of someone who is
looking at it on github.com and thought that the latest Readme.md would be
visible to them there, hence moved it to the bottom.

This text as well as the badges seemed to be taking up lots of space when
looking at the page, right on top. Maybe there is a different way to
distinguish between the two sets of badges, without the huge headings.

Please feel free to change the PR and squash it too - I did't realize that
multiple changes I made in the fork will all become part of the PR.

ShreeDevi
____________________________________________________________
भजन - कीर्तन - आरती @ http://bhajans.ramparivar.com

**Comment by egorpugin** [comment]
Badges look ugly now.
https://github.com/Shreeshrii/tesseract/blob/master/README.md

Current form is better.

**Comment by Shreeshrii** [comment]
@egorpugin I have reverted the change for badges and updated link for 3.05.01 release. Please make any other changes that you see fit.

Also, please add your name as well as those of other maintainers/members with write access to the repo.

**Pull request merged**